### PR TITLE
Migrate chosen devteam tools

### PR DESCRIPTION
**Pull request by yhoogstrate**
Hi @martenson,

I've updated the indentation, added a few macro's and added `argument=`-tags in FreeBays. I hope you like the changes.

all best,

Youri

### CODE REVIEW

File: tools/freebayes/leftalign.xml [hunk 1]
```diff
@@ -49,7 +51,8 @@
                 <param name="ref_file" type="data" format="fasta" label="Using reference file" />
```
Comment (nsoranzo) [inline tools/freebayes/leftalign.xml:51]: Add ` argument="--fasta-reference"` also here?

File: tools/freebayes/freebayes.xml [hunk 2]
```diff
@@ -287,93 +299,91 @@
...
                             <param name="input_variant_type_selector" type="select" label="Provide variants file">
                                 <option value="do_not_provide" selected="True">Do not provide</option>
                                 <option value="provide_vcf">Provide VCF file</option>
                             </param>
-                            <when value="do_not_provide">
-                                <!-- Do nothing here -->
-                            </when>
+                            <when value="do_not_provide" /><!-- Do nothing here -->
                             <when value="provide_vcf">
                                 <param name="input_variant_vcf" type="data" format="vcf_bgzip" label="Use variants reported in VCF file as input to the algorithm">
```
Comment (nsoranzo) [inline tools/freebayes/freebayes.xml:351]: Add `argument="--variant-input"`

File: tools/freebayes/freebayes.xml [hunk 3]
```diff
@@ -953,13 +963,5 @@ The initial version of the wrapper was produced by Dan Blankenberg and upgraded
...
-    <citations>
-        <citation type="bibtex">@misc{1207.3907,
-Author = {Erik Garrison},
-Title = {Haplotype-based variant detection from short-read sequencing},
-Year = {2012},
-Eprint = {arXiv:1207.3907},
-url = {http://arxiv.org/abs/1207.3907},
-}</citation>
-    </citations>
+    <expand macro="citations" />
```
Comment (nsoranzo) [inline tools/freebayes/freebayes.xml:966]: Also the sentence few lines above
```
For the underlying tool, please cite `Erik Garrison and Gabor Marth. Haplotype-based variant detection from short-read sequencing &lt;http://arxiv.org/abs/1207.3907&gt;`_.
```
can be removed.

**Comment by martenson** [comment]
xref: https://github.com/galaxyproject/tools-iuc/pull/1109

@yhoogstrate you have push rights to `migrate-chosen-devteam-tools` branch on martenson/tools-iuc. Please don't hesitate to push there directly.

note: sorry for delay, this repo is not watched :/

**Comment by martenson** [comment]
@nsoranzo I will merge this if you approve, could you please push your other comments directly to my branch?

**Comment by nsoranzo** [comment]
@martenson Sure!

**Pull request merged**